sections/semantics-document-metadata.include

```diff
@@ -680,14 +680,13 @@
     <a>supported tokens</a> are the keywords defined in
     <a>HTML link types</a> which are allowed on <code>link</code> elements, impact
     the processing model, and are supported by the user agent. The possible supported tokens are
-    <code>alternate</code>, <code>icon</code>, <code>prefetch</code>,
-    <code>stylesheet</code> and <code>next</code>.
+    <code>alternate</code>, <code>author</code>, <a>dns-prefetch</a>, <code>help</code>, <code>icon</code>, <code>license</code>, <code>next</code>, <a>preconnect</a>, <a>prefetch</a>, <a>prerender</a>, <code>prev</code>, <code>search</code>, and <code>stylesheet</code>.
```

Comment:
This section includes the caveot that the list includes those tokens that **impact the processing model**. I don't believe that:
* author
* help
* license
* search
* prev/next (I think is was a bug that `next` was in the list before.)